tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Setup/PackageContentTests.cs

```diff
@@ -62,7 +65,7 @@ private static IEnumerable<string> GetPackageContents(string vsixName)
             
             using var archive = ZipFile.OpenRead(vsixPath);
             
-            return archive.Entries.Select(entry => entry.FullName);
+            return archive.Entries.Select(entry => entry.FullName).OrderBy(fn => fn);
```

Comment:
The test was changed so that the file entries are now in alphabetical order! The order of the files before was based on the repacking process that RepoToolset did during the signing phase. When removing that signing process, this file order was no longer in a determined order. Thus, we explicitly order the entries here, and I've updated the txt files to be in alphabetical order to match.